web/crux/.env.example

```diff
@@ -9,6 +9,7 @@ CRUX_UI_URL=http://localhost:8000
 # Port settings
 GRPC_AGENT_PORT=5000
 HTTP_API_PORT=1848
+METRICS_API_PORT=1849
```

Comment:
I'm thinking that we should use 1956 for the Metrics port; it would be more appropriate. What do you think?

Reply:
Sure, why not. :D